src/ia-item-user-lists.ts

```diff
@@ -111,6 +121,21 @@ export class IaItemUserLists extends LitElement {
     });
   }
 
+  updated(changed: PropertyValues): void {
+    if (changed.has('baseHost') && this.baseHost) {
+      if (changed.get('baseHost')) {
+        this.reloadService();
+      }
+    }
```

Comment:
I think this may be equivalent?

```suggestion
    if (changed.get('baseHost') && this.baseHost) {
      this.reloadService();
    }
```

Reply:
no, not quite. it must have already a basehost before changing value. that's handled int 1st updated, whose main value there is that it's explicit. we will make another call if otherwise. thanks for asking, I thought about it, too. will keep eye out when we can streamline this better.